index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Swiggy</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <img src="images/unnamed.png">
+    </header>
+    <section>
+        <header>
+        <img src="https://images.unsplash.com/photo-1625220194771-7ebdea0b70b9?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2070&q=80">
+        <p>Punjabi Momos</p>
+    </header>
+    <article>
+        <div class="navbar">
+            <ul>
+                <li class="heading">Recommended</li>
+                <li>Chandni Chowk Ki Biryani</li>
+                <li>Naughty Momos</li>
+                <li>Naughty Momo Roll</li>
+            </ul> 
+        </div>
+        <div class="menu">
+            <h1>Recommended</h1>
+            <h6>24 ITEMS</h6>
+            <ul>
+                <li>
+                    <div class="dish">
+                    <img src="images/240px-Indian-vegetarian-mark.svg.png">
```

Comment:
Always give an alt tag to the images - it helps assistive technologies read this out to the user and also serves as a fallback in case the image does not load